reladomo-tour/tour-examples/auditonly-bank/src/test/java/auditonlybank/api/AuditOnlyChainingInAction.java

```diff
@@ -176,4 +198,20 @@ private void balanceAsOfJan17(int accountId)
         */
         assertEquals(100, (int) account.getBalance());
     }
+
+    /*
+        This method explicitly sets the transaction's processing time to the supplied value.
+        You should rarely have to do this in production.
+
+        It is being done here just to simulate a database operation happening on a specific date,
+        so that the output of thus program matches the narrative in the Reladomo tour docs.
+
+        Without this, the processing time gets set to the time of running this program.
```

Comment:
Add text:
Calling this method in production code violates the auditability of the data, which is the only reason the processingDate exists in the first place.